<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Global configuration</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-30</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Build Settings</string>
    </property>
    <widget class="QLabel" name="incredibuildLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use incredibuild:</string>
     </property>
    </widget>
    <widget class="QLabel" name="sdkDirLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Unique SDK dir:</string>
     </property>
    </widget>
    <widget class="QLabel" name="buildDirLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Unique build dir:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="incredibuildCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="sdkLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="buildLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="skdDirButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse SDK dir ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buildDirButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse Build dir ...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="envGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Environnement Settings</string>
    </property>
    <widget class="QLabel" name="envPathLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="envPathLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>skdDirButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWidget</receiver>
   <slot>onBrowseSdkDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildDirButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigWidget</receiver>
   <slot>onBrowseBuildDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ConfigWidget</receiver>
   <slot>onButtonBoxClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBrowseSdkDir()</slot>
  <slot>onBrowseBuildDir()</slot>
  <slot>onButtonBoxClicked(QAbstractButton*)</slot>
 </slots>
</ui>
